doc/source/index.rst

```diff
@@ -3,21 +3,23 @@ Zero to JupyterHub with Kubernetes
 
 `JupyterHub`_ allows users to interact with a computing environment through a
 webpage. As most devices have access to a web browser, JupyterHub makes it is
-easy to provide and standardize the computing environment of a group of people
+easy to provide and standardize the computing environment for a group of people
 (e.g., for a class of students or an analytics team).
 
-This project will help you set up your own JupyterHub on a cloud and leverage
-the clouds scalable nature to support large groups of users. Thanks to
+This project will help you set up your own JupyterHub on a cloud/on-prem k8s environment and leverage
+its scalable nature to support a large group of users. Thanks to
 :term:`Kubernetes`, we are not tied to a specific cloud provider.
 
-Note that this project is under active development so information and tools may
-change. *You can be a part of this change!* If you see anything that is incorrect
-or have any questions, feel free to reach out in the `gitter chat
-<https://gitter.im/jupyterhub/jupyterhub>`_ or create an issue at the `issues
-page`_. If you have tips or deployments that you would like to share, see
-:ref:`community-resources`.
+.. note::
+
+   This project is under active development so information and tools may
+   change. *You can be a part of this change!* If you see anything that is incorrect
+   or have any questions, feel free to reach out in the `gitter chat
+   <https://gitter.im/jupyterhub/jupyterhub>`_ or create an issue at the `issues
+   page`_. If you have tips or deployments that you would like to share, see
+   :ref:`resources-and-references`.
```

Comment:
We should mention Discourse too. 
(I'm not sure of the indentation of the suggestion here is ok, though :thinking: )
```suggestion
   change. *You can be a part of this change!* If you see anything that is incorrect, please create an issue at the
   `issues page`_. If you have any questions, feel free to post on the community `Discourse forum
   <https://discourse.jupyter.org/>`_ or reach out in the `Gitter chat <https://gitter.im/jupyterhub/jupyterhub>`_ or . If you
   have tips or deployments that you would like to share, see :ref:`resources-and-references`.
```

Reply:
Yes! Great addition, I did not notice that Discourse was not mentioned here.